### PR TITLE
Hero section 1 #3

### PR DESCRIPTION
**Pull request by Kaifeng-Sun**
image

### CODE REVIEW

File: src/styles/global-font-style.scss [hunk 1]
```diff
@@ -59,4 +59,39 @@
 .font_navigator_logo {
   @include subtitle(600);
   line-height: 20px;
+}
+
+.font_hero_title {
+  @include h1-headline(700);
+  color: #fff;
+  text-shadow: 0px 4.1504716873168945px 21.897809982299805px rgba(0, 0, 0, 0.25);
```
Comment (Prom1s1ngYoung) [inline src/styles/global-font-style.scss:67]: I am just curious why are there so many digits after the decimal point?

File: src/styles/global-font-style.scss [hunk 2]
```diff
@@ -59,4 +59,39 @@
...
+  color: #fff;
+  font-feature-settings: "clig" off, "liga" off;
+  font-family: Raleway;
+  line-height: 24px;
+}
+
+.font_hero_background {
+  color: rgba(254, 254, 254, 0.07);
+  font-family: Open Sans;
+  font-size: 350.104px;
```
Comment (Prom1s1ngYoung) [inline src/styles/global-font-style.scss:85]: Same here

**Comment by Prom1s1ngYoung** [comment]
Others look good to me.

File: src/styles/global-font-style.scss (hunk 1 above)
Comment (Kaifeng-Sun) [reply src/styles/global-font-style.scss:67]: Those numbers are copied from figma.

Comment (tatoMa) [reply src/styles/global-font-style.scss:67]: > Those numbers are copied from figma.

So you did not attend and watch the QA session? There is no way to use a decimal number pixel in CSS.

Comment (tatoMa) [reply src/styles/global-font-style.scss:67]: Round it to a closer even number.

Comment (Kaifeng-Sun) [reply src/styles/global-font-style.scss]: Solved

Comment (Prom1s1ngYoung) [reply src/styles/global-font-style.scss]: I think the digits after the decimal point will affect very little to most of us. I searched Google that shows the majority of browsers now ignore the decimals.

File: src/styles/global-font-style.scss [hunk 3]
```diff
@@ -80,9 +80,9 @@
 }
 
 .font_hero_background {
-  color: rgba(254, 254, 254, 0.07);
```
Comment (Prom1s1ngYoung) [inline src/styles/global-font-style.scss]: The 4th arg in the rgba means alpha channel, it should be a decimal around 0 to 1. You can check the UAT [taotify-website.vercel.app](https://taotify-website.vercel.app/), the background font is now missing, and I think you should turn it back.

Comment (Kaifeng-Sun) [reply src/styles/global-font-style.scss]: And also the background color. Please check one more time thx.

**Review by tatoMa — CHANGES_REQUESTED** [review]
There are differences between our Figma design and our frontend, please check it again.

image

**Review by tatoMa — APPROVED** [review]

**Review by tatoMa — APPROVED** [review]
LGTM

**Pull request merged**